recipes/hyphy/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.3.14" %}
+{% set version = "2.5.0" %}
 {% set sha256 = "9e6c817cb649986e3fe944bcaf88be3533e7e62968b9a486c719e951e5ed1cf6" %}
```

Comment:
You need to change the SHA sum.  It should be 615ae5bd4e786f9aa41722ed8570107890a35e5a07c95a77af71763613f564ab, which I got from running the following on a Linux machine:

```
wget https://github.com/veg/hyphy/archive/2.5.0.tar.gz
sha256sum 2.5.0.tar.gz
```

Reply:
Thank you @molpopgen!